service/src/main/java/io/camunda/service/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -50,6 +52,7 @@ public SearchQuery toSearchQuery(final UserTaskFilter filter) {
     final var assigneesQuery = getAssigneesQuery(filter.assignees());
     final var stateQuery = getStateQuery(filter.states());
     final var tenantQuery = getTenantQuery(filter.tenantIds());
+    final var priorityQuery = getComparableFilter(filter.priority(), "priority");
```

Comment:
❓ Is this field already included on "tasklist-task" indice?

Reply:
Will be when the time to merge this PR comes